docs/sphinx/source/index.rst

```diff
@@ -60,11 +60,54 @@ Contents
    :maxdepth: 2
    
    self
+   package_overview
    whatsnew
+   usage
+   modules
+   classes
    comparison_pvlib_matlab
    variables_style_rules
-   pvlib
 
+   
+Contents (potential new ToC)
+============================
+
+* What’s New
+* Overview: Why pvlib?
+    * Introduction
+    * Modeling paradigms
+    * User extensions
+    * What it is not
+    * Comparison with PVLIB_MATLAB
+* Installation
+* Examples
+    * Using the principal components
+        * atmosphere.py tutorial
+        * solarposition.py tutorial
+        * irradiance.py tutorial 
+    * Specific datasets
+        * TMY tutorial 
+        * TMY data and diffuse irradiance model 
+        * TMY to Power Tutorial 
+    * System Modelling
+        * pvsystem tutorial
+        * Tracking
+        * Modeling multiple systems with system losses
+    * Validations
```

Comment:
Could be "Validations & Comparisons"

In the "SAPM" notebook, the different models are compared (= validated against each other).
This a strength of pvlib as compared to other tools albeit SAM & PVSyst can also do this...